docs/literate/man/utils.jl

```diff
@@ -1,13 +1,111 @@
 # # Utilities
 
 # ## Gates
+# ```@docs
```

Comment:
This 
```
│ ```@docs
│ QuantumUtils.GATES
│ ```
```
needs to be removed. It's causing the CI doc error.

Reply:
Applied the Fix with recent commit.